File: tmp-mei/fgenjs5030319945734616766/base/ControlFlowReturn_1.js
import { base$$ControlFlow_1 } from "../base/ControlFlow_1.js";
import * as rt from "../rt/main.js";

export class base$$ControlFlowReturn_1 extends base$$ControlFlow_1 {
  static $self = new base$$ControlFlowReturn_1();

  match$mut(m_m$) {
  return base$$ControlFlowReturn_1.match$mut$fun(m_m$, this);
}


value$mut() {
  return base$$ControlFlowReturn_1.value$mut$fun(this);
}

  static match$mut$fun(m_m$, $this) {
  return (function() {
  console.error("Program aborted at:\n" + new Error().stack);
  if (typeof process !== "undefined") process.exit(1);
  else throw new Error("Program aborted");
})()
;
}

    static value$mut$fun($this) {
  return (function() {
  console.error("Program aborted at:\n" + new Error().stack);
  if (typeof process !== "undefined") process.exit(1);
  else throw new Error("Program aborted");
})()
;
}

}

File: tmp-mei/fgen4255027576711118260/base/ControlFlowReturn_1.java
package base;
public interface ControlFlowReturn_1 extends base.ControlFlow_1,base.Sealed_0{
ControlFlowReturn_1 $self = new ControlFlowReturn_1Impl();
Object match$mut(base.ControlFlowMatch_2 m_m$);

Object value$mut();
static Object match$mut$fun(base.ControlFlowMatch_2 m_m$, base.ControlFlowReturn_1 $this) {
  return (switch (1) { default -> {
  System.err.println("Program aborted at:\n"+java.util.Arrays.stream(Thread.currentThread().getStackTrace()).map(StackTraceElement::toString).collect(java.util.stream.Collectors.joining("\n")));
  System.exit(1);
  yield (Object) null;
}})
;
}

static Object value$mut$fun(base.ControlFlowReturn_1 $this) {
  return (switch (1) { default -> {
  System.err.println("Program aborted at:\n"+java.util.Arrays.stream(Thread.currentThread().getStackTrace()).map(StackTraceElement::toString).collect(java.util.stream.Collectors.joining("\n")));
  System.exit(1);
  yield (Object) null;
}})
;
}
}